TabnineSelfHosted/src/main/java/com/tabnineSelfHosted/statusBar/TabnineSelfHostedStatusBarWidget.kt

```diff
@@ -123,13 +123,9 @@ class TabnineSelfHostedStatusBarWidget(project: Project) :
         if (!hasCloud2UrlConfigured()) {
             return "Tabnine Enterprise: Set your Tabnine URL"
         }
-        val cloudConnectionHealthStatus = getCloudConnectionHealthStatus()
-        val userInfo = getLastUserStatus()
-
-        if (cloudConnectionHealthStatus == null) {
-            return "Tabnine Enterprise: Initializing"
-        }
+        val cloudConnectionHealthStatus = getCloudConnectionHealthStatus() ?: return "Tabnine Enterprise: Initializing"
```

Comment:
We had issues in the past with the plug-in showing endless initializing message. How can we make sure we're not going back to that mess?

Reply:
Do we prefer tomshow wrong status?

Reply:
no, but will it keep initializing or fail?

Reply:
it will fail, as in some point it will move from "null" to `fail`

Reply:
👍